README.md

```diff
@@ -0,0 +1,148 @@
+# Overview
+
+GD-Training-Project is an implementation of the online store back-end functionality.
+Checking the correctness of the developed functionality can be done via Postman.
+The main functionality that is implemented in the project is working with a database, namely
+
+Product management:
+
+- adding a product
+
+- product price update
+
+- receiving all products stored in the database
+
+- receiving products by their name or unique id.
+
+## Summary
+
+  - [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installing and Deployment](#installing-and-deployment)
+
+
+
+## Getting Started
+
+**Requirements:**
+- OS: Windows, macOS, Linux
+- JDK 8 and over
+- Docker + MySql 8.0.15 (and over) container
+- Maven 3.6.3 and over
+
+
+
+
+For use and run application it is needed to clone the project to the local machine:
+
+    git clone https://github.com/an-kiian/GD-Training-Project
+
+## Prerequisites
+
+**Maven**
+
+For install maven you need to download package from the official [resource](https://maven.apache.org/download.cgi) and [install](https://mkyong.com/maven/how-to-install-maven-in-windows/) it.
+
+**IntelliJ IDEA**
+
+- download IntelliJ IDEA [Ultimate version](https://www.jetbrains.com/idea/promo/ultimate/?gclid=CjwKCAjwmMX4BRAAEiwA-zM4JogGjxxekfBMsDbtXyjyMEA_iB-_ii3aldVTHvjOrl7fOgTLPl3d8RoCVvUQAvD_BwE). 
+- add Lombok plugin according to the [guideline](https://projectlombok.org/setup/intellij).
+- add your username and password in `main/resources/application.properties` file of project :
+
+      spring.datasource.username=[user_name]
+      spring.datasource.password=[password]
+
+
```

Comment:
I would like to have an instruction to run application without downloading Intelliji